cwl/facets-suite-workflow.cwl

```diff
@@ -426,6 +426,21 @@ steps:
     out:
       [ output_file, failed_txt, stdout_txt, stderr_txt ]
 
+  merge_maf:
+    run: update_cBioPortal_data.cwl
+    in:
+      pair_id: pair_id
+      subcommand:
+        valueFrom: ${ return "merge_mafs"; }
+      input_file: check_label_maf_normal/output_file
```

Comment:
yea I think this is supposed to be the portal maf file `data_mutations_extended.txt` file